### PR TITLE
[Bug Fix] Eat connection error in an info handler.

### PR DESCRIPTION
**Pull request by 3Hren**
When the engine is overloaded it can take a long time to handle an info request. The user can decide to disconnect, which results for us in "session not connected" error, after trying to write into the deferred. This exception is propagated up to event loop't thread and terminates it.

In this case here we can't do anything with it except eating an exception and to log it.

If you see more elegant solution, tell me about it.

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,10 @@
+cocaine-core (0.12.0.2) unstable; urgency=low
```
Comment (Kobolog) [inline debian/changelog]: 0.12.0.2?

Comment (3Hren) [inline debian/changelog]: We already have 0.12.0.1 in https://github.com/cocaine/cocaine-core/pull/165/files#diff-f758558c8bab6e325c78465f6a7fa64bR132

Comment (Kobolog) [inline debian/changelog]: This is because RPM is too dumb to understand 0.12.0.0.

Comment (3Hren) [inline debian/changelog]: Even 0.12.0.0-1?

**Comment by Kobolog** [comment]
My point is why did you bump version at all? We never had a proper release yet.

**Pull request merged**